packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -53,10 +53,7 @@ export const certificateTransactionsWithEpochs = (
   slotEpochCalc$: Observable<SlotEpochCalc>,
   certificateTypes: Cardano.CertificateType[]
 ): Observable<TxWithEpoch[]> =>
-  combineLatest([
-    transactionsWithCertificates(transactionsTracker.history.outgoing$, certificateTypes),
-    slotEpochCalc$
-  ]).pipe(
+  combineLatest([transactionsWithCertificates(transactionsTracker.history$, certificateTypes), slotEpochCalc$]).pipe(
```

Comment:
Is this ok? We don't have `transactionsTracker.history.outgoing$` anymore

Reply:
Looking at this now, it wasn't 100% correct to start with. This change made it a little bit worse, but let's address it separately in ADP-1719

The issue is that this is then used [here](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/wallet/src/services/DelegationTracker/RewardsHistory.ts#L37) (and possibly a few other places), which does not check `rewardAccount` in the certificate, so if someone sends you some coin and delegates in the same tx, it will be a bug.

Reply:
> let's address it separately in ADP-1719

Should I leave it like this then?